index.js

```diff
@@ -1,3 +1,146 @@
+    technologies: ['Html', 'Bootstrap', 'Ruby'],
+    linkToLiveVersion: 'https://summyalena.github.io/Website-Portfolio/',
+    LinkToSource: 'https://github.com/summyalena/Website-Portfolio',
+  },
+  card5: {
+    id: 5,
+    title: 'Professional Art Printing Data',
+    description: "A daily selection of privately personalized reads no accounts or sign-ups required. has been the industry standard Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    featuredImage: './images/popimage.png',
+    technologies: ['Html', 'Bootstrap', 'Ruby'],
+    linkToLiveVersion: 'https://summyalena.github.io/Website-Portfolio/',
+    LinkToSource: 'https://github.com/summyalena/Website-Portfolio',
+  },
+  card6: {
+    id: 6,
+    title: 'Professional Art Printing Data',
+    description: "A daily selection of privately personalized reads no accounts or sign-ups required. has been the industry standard Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    featuredImage: './images/popimage.png',
+    technologies: ['Html', 'Bootstrap', 'Ruby'],
+    linkToLiveVersion: 'https://summyalena.github.io/Website-Portfolio/',
+    LinkToSource: 'https://github.com/summyalena/Website-Portfolio',
+  },
+};
+
+const cardSection = document.querySelector('.cardworks');
+
+Object.keys(cardWorks).forEach((cards) => {
+  if (Object.hasOwnProperty.call(cardWorks, cards)) {
+    const card = cardWorks[cards];
+    cardSection.innerHTML += `
+  <div class="card-images">
+    <div class="cardwork-container">
+      <div class=" cardwork cardwork1">
+        <div class="cardcon">
+          <h3>${card.title}</h3>
+          <p>
+          ${card.description.length > 90 ? card.description.substring(0, 90) : card.description}
+          </p>
+          <div class="spancon">
+            <span class="span">HTML</span>
+            <span class="span">CSS</span>
+            <span class="span">JAVASCRIPT</span>
+          </div>
+        </div>
+        <div class="button">
+          <a href="" id= "${card.id}" class="see" >See Project</a>
+        </div>
+      </div>
+    </div>
+  </div>`;
+  }
+});
+
+const projectPopup = document.querySelector('.project-popup');
+const popButton = document.querySelectorAll('.see');
+
+popButton.forEach((popup) => {
+  popup.addEventListener('click', (e) => {
+    e.preventDefault();
+    Object.keys(cardWorks).forEach((cards) => {
```

Comment:
- [ ] The mobile version of the pop-up modal is different in design when compared to the original template. Kindly fix it.

Your design
![image](https://user-images.githubusercontent.com/81932478/180247473-96478970-fc8e-4fcf-847a-56f1e713c160.png)

Provided design
![image](https://user-images.githubusercontent.com/81932478/180248573-bd9dae21-c07e-4344-afbf-ef7ef8f3d2f7.png)